applications/nrf5340_audio/src/audio/audio_system.c

```diff
@@ -95,28 +95,26 @@ static void audio_gateway_configure(void)
 		ERR_CHK_MSG(-EINVAL, "No codec selected");
 	}
 
+	/* For multi-channel currently only left and right front are supported */
+	sw_codec_cfg.encoder.audio_loc =
+		BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT;
```

Comment:
Can be moved to the `else` below. But we could also remove the else and just add a return in the ´if´ (though you would have to add a second LOG_INF.